fmfexporter/adapters/polarion/polarion_reporter.py

```diff
@@ -14,27 +15,28 @@
 from fmfexporter.adapters.polarion.polarion_test_case import PolarionTestCase
 from fmfexporter.adapters.polarion.utils.polarion_config import PolarionConfig
 
-
 LOGGER = logging.getLogger(__name__)
 urllib3.disable_warnings()
 
 
 class PolarionReporter(object):
-
     """
     Provides methods for submitting test cases (from PolarionTestCase) objects into Polarion.
     """
+
     def __init__(self, config_file):
         self.config = PolarionConfig(config_file)
         self.headers = {'Accept': 'application/json'}
         self.auth = HTTPBasicAuth(self.config.username(), self.config.password())
+        self.testcase = None  # type: PolarionTestCase
```

Comment:
Why is it needed?

Reply:
This is the imported url test case work item (constructed url `http://polarion/testcase-work-teim?AMQ-12345`)

Reply:
At a glance it makes more sense to pass the tc as an argument, as the same reporter instance is used for multiple tcs.

Reply:
Fixed by removal after rebase after Jakub's changes.